### DISCUSSION TITLE
How to hinder hyperlinks within third-party applications in SEB, that open URL's outside the software?

### DISCUSSION THREAD

**Discussion by ElNaRex** [SafeExamBrowser/seb-win-refactoring#588] [Q&A] [answered]
So we have enabled the pupils to make use of a third-party dictionary inside Safe Exam Browser on Windows (in the "Applications" tab). However, the hyperlinks inside the third-party application (within SEB) still opens up outside of SEB. The URL is for example a "New version available" link inside the dictionary application, which opens the default browser on the computer outside SEB. 

I have tried to hinder the hyperlinks by doing the following, but to no prevail:
> In the "Network" tab, click the "+" button in the "Filter" section to add a new blocked URL pattern. Enter a wildcard pattern to block all other URLs (e.g., https://* and http://*).

Is there something I've overlooked? Any help would be highly appreciated!

**Comment by dbuechel** [comment, marked as answer]
SEB cannot control the functionality of third-party applications. The network tab relates only to SEB's internal browser component, not to the entire network traffic on a computer.

**Reply by ElNaRex** [reply]
Thank you very much for a swift reply, @dbuechel! Highly appreciate it :)

**Comment by danschlet** [comment]
@ElNaRex what I don't understand, why do these hyperlinks open up outside of SEB? Are you using SEB without its kiosk mode (with the debug mode)?
